proto/viam/app/data/v1/data.proto

```diff
@@ -80,6 +80,14 @@ message Filter {
   repeated string org_ids = 11;
   repeated string mime_type = 12;
   CaptureInterval interval = 13;
+  TagsOption tags_option = 14;
+}
+
+enum TagsOption {
+  TAGS_OPTION_UNSPECIFIED = 0;
+  TAGS_OPTION_MATCHING = 1;
```

Comment:
Would be helpful to have a short comment for each of these options

Reply:
Done, let me know if they're clear